typescript/sdk/src/middleware/liquidity-layer/LiquidityLayerApp.ts

```diff
@@ -253,10 +266,13 @@ export class LiquidityLayerApp extends HyperlaneApp<
       return;
     }
 
+    console.log(`Attempt Circle message delivery`, JSON.stringify(message));
```

Comment:
Think I've mentioned it before but it's strange to see console logging here but `debug` lib everywhere else in the SDK

Reply:
Noted, but this is effectively deprecated so lets keep as is